### ISSUE TITLE
Uppercase and lowercase

### ISSUE THREAD

**Issue by axeloz** [timber/timber#1001]
Hello Guys,

I'm getting a little bit confused here. Last week, you release Timber 1.0.1. This version works on a Mac in local but fails on a Linux PREPROD server. After investigation, we notice that some files are named in lowercase when called with uppercase naming. For instance: the "timber.php" class or the "cache" folder. 

Quickly after that, you release Timber 1.0.2 that fixes all the uppercase issues but we don't get the chance to upgrade immediatly.

First thing this morning, we want to upgrade to 1.0.2 but surprise, you have upgraded to 1.0.3. And all the case issues have reappeared. 

How comes? Linux servers aren't tolerant at all about file case naming. When using namespace "Timber/Timber", you must make sure that folders and filename respect the same case. 

Thanks a lot, 

Kind regards

Axel

**Comment by connorjburton** [comment]
Hi,

As far as I can see in our source and on the release ZIPs everything that is namespaced is CamelCased. How are you receiving these updates?

We haven't done any fixes in `1.0.2` regarding file/folder naming AFAIK

**Comment by axeloz** [comment]
Hello @connorjburton , thanks for your reply. 

After further investigation, I think I got it. We upgraded last week from 0.22 (which uses lib/timber.php) to 1.0.1 (which uses lib/Timber.php). On a Mac, it seems the Git didn't see any difference between timber.php and Timber.php so the file was never renamed when pushing. What works on the Mac doesn't on Linux. 

Sorry about the ticket, the problem is on my side. 
Thanks a lot,

Axel
